src/core/types.h

```diff
@@ -434,6 +434,7 @@ class Box {
     // Add a no-op constructor to make sure that we don't zero-initialize cls
     Box() {}
 
+    void* operator new(size_t size, BoxedClass* cls, size_t nitems) __attribute__((visibility("default")));
```

Comment:
yeah I think so. I didn't really notice that these were here... who uses them? I had assumed that all subclasses of `Box` needed to specify their own `operator new`